components/AppHeader.vue

```diff
@@ -16,15 +16,26 @@
         <a
           href=""
           class="ml-16 text-2xl"
+          :class="{ 'text-amber-400': selectedLink === 'admin' }"
```

Comment:
String comparison. Should use an enum :)

Reply:
Using an enum now